docs/standard/garbagecollection/implementing-dispose.md

```diff
@@ -14,28 +14,28 @@ ms.assetid: eca6cdc3-6a14-4296-86fb-1eb2f21455b0
 
 # Implementing a dispose method
 
-You implement a [Dispose](@System.IDisposable#System_IDisposable_Dispose) method to release unmanaged resources used by your application. The .NET garbage collector does not allocate or release unmanaged memory. 
+You implement a [Dispose](xref:System.IDisposable#System_IDisposable_Dispose) method to release unmanaged resources used by your application. The .NET garbage collector does not allocate or release unmanaged memory. 
 
 The pattern for disposing an object, referred to as a dispose pattern, imposes order on the lifetime of an object. The dispose pattern is used only for objects that access unmanaged resources, such as file and pipe handles, registry handles, wait handles, or pointers to blocks of unmanaged memory. This is because the garbage collector is very efficient at reclaiming unused managed objects, but it is unable to reclaim unmanaged objects.
 
 The dispose pattern has two variations:
 
-* You wrap each unmanaged resource that a type uses in a safe handle (that is, in a class derived from [System.Runtime.InteropServices.SafeHandle](@System.Runtime.InteropServices.SafeHandle)). In this case, you implement the [IDisposable](@System.IDisposable) interface and an additional `Dispose(Boolean)` method. This is the recommended variation and doesn't require overriding the [Object.Finalize](@System.Object#System_Object_Finalize) method. 
+* You wrap each unmanaged resource that a type uses in a safe handle (that is, in a class derived from [System.Runtime.InteropServices.SafeHandle](xref:System.Runtime.InteropServices.SafeHandle)). In this case, you implement the [IDisposable](xref:System.IDisposable) interface and an additional `Dispose(Boolean)` method. This is the recommended variation and doesn't require overriding the [Object.Finalize](xref:System.Object#System_Object_Finalize) method. 
```

Comment:
when you don't want the fully qualified name such as in `[IDisposable](xref:System.IDisposable)`, you can just use @System.IDisposable so you don't have to worry about link text.